robot/demos/SkidSteerServos/SkidSteeringDC.ino

```diff
@@ -0,0 +1,93 @@
+#include <ArduinoBlue.h>
+#include <SoftwareSerial.h>
+#include <Servo.h>
+
+int prevThrottle = 49;
+int prevSteering = 49;
+int throttle, steering;
+// Initialize Left & Right Velocity variables to Rest
+int VR = 127;
+int VL = 127;
+SoftwareSerial bluetooth(7, 8);
+
+ArduinoBlue phone(bluetooth);
+
+void setup() {
+  // Attach Servos to pins:
+  // Right side servos
+  DC_PIN_1 = 9;
```

Comment:
```suggestion
  DC_PIN_FRONT_RIGHT = 9;
  DC_PIN_MIDDLE_RIGHT = 10;
  DC_PIN_BACK_RIGHT = 11;
  
  // Left side DC motors
  DC_PIN_FRONT_LEFT = 3;
  DC_PIN_MIDDLE_LEFT = 5;
  DC_PIN_BACK_LEFT = 6;
```

Reply:
Just a suggestion. If you hate my nomenclature you can pick your own or whatever is used in the litterature for 6-wheeled vehicles